reference/api/indexes.md

```diff
@@ -64,15 +64,19 @@ Get information about an [index](/learn/core_concepts/indexes.md).
 
 <RouteHighlighter method="POST" route="/indexes"/>
 
-Create an [index](/learn/core_concepts/indexes.md).
+Create an [index](/learn/core_concepts/indexes.md). This route accepts two arguments: `index_uid` and `primaryKey`.
 
-This route takes as parameter an unique `uid` and **optionally** the [primary key](/learn/core_concepts/indexes.md#primary-key).
+`index_uid` is mandatory. It should be a unique string identifying your new index.
+
+`primaryKey` is optional and indicates the field containing your documents' [primary key](/learn/core_concepts/indexes.md#primary-key). If you do not supply a value for `primaryKey`, MeiliSearch will do its best to infer it once you add your first document to the index.
```

Comment:
Chibi MeiliSearch is great. I name you head vice-director of branding and marketing-adjacent subjects.

Jokes aside, though I do get your point, making primary key inferral come across as unreliable was actually sort of intentional. My understanding is that we are not supposed to encourage people to rely too much on inferring the primary key since we cannot guarantee it will work.